ui_framework/components/table/landing_page_table/landing_page_tool_bar_footer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { KuiToolBarPagerSection, KuiToolBarFooterSection, KuiToolBarText } from '../../tool_bar';
+
+export function LandingPageToolBarFooter({ selectedItemsCount, pagerState, onPreviousPage, onNextPage }) {
+  function getSelectedItemsSection() {
```

Comment:
I avoided it because it was so specific, being a footer section (`KuiSelectedItemsFooterSection`), but sg, I can move out.